packages/hub/src/lib/oauth-handle-redirect.ts

```diff
@@ -1,28 +1,31 @@
 import { HUB_URL } from "../consts";
 import { createApiError } from "../error";
 
+export interface UserInfo {
+	sub: string;
+	name: string;
+	preferred_username: string;
```

Comment:
If we end up going this way, it might be helpful to add a comment stating that these come from OpenId directly and therefore don't follow our usual conventions.

Reply:
Good point, commented the fields in https://github.com/huggingface/huggingface.js/pull/1050/commits/be017493ec64739347b0bf6ad971b28bccfb00f4